data/mods/BlazeIndustries/furniture_and_terrain/blaze_appliances.json

```
@@ -123,5 +123,20 @@
       { "item": "food_processor", "charges": 0, "prob": 50 },
       { "item": "press", "prob": 50 }
     ]
+  },
+  {
+    "type": "vehicle_part",
+    "id": "afs_washing_machine",
```

```suggestion
    "id": "blaze_washing_machine",
```
I'm not sure why you would give this the afs prefix when it didn't originate in Aftershock?

---

That's a good question.

Seems to be residual code that Blaze used and copied from aftershock. All of the vehicle stuff is labelled similarly.
I could go and address that in this PR I guess.